pallets/vanchor/Cargo.toml

```diff
@@ -44,6 +45,7 @@ ark-crypto-primitives = { version = "^0.3.0", features = ["r1cs"], default-featu
 ark-ec = { version = "^0.3.0", default-features = false }
 ark-ff = { version = "0.3.0", default-features = false, features = ["asm"] }
 ark-groth16 = { git = "https://github.com/arkworks-rs/groth16", rev = "765817f", default-features = false, features = ["parallel"] }
```

Comment:
Should this be there?

Reply:
It is in the `dev-dependencies`. But no, we can remove it, will clean up and push the changes.